core/src/main/java/com/amplifyframework/datastore/DataStoreChannelEventName.java

```diff
@@ -52,7 +52,22 @@
     /**
      * The remote synchonization processes stopped.
      */
-    REMOTE_SYNC_STOPPED("remote_sync_stopped");
+    REMOTE_SYNC_STOPPED("remote_sync_stopped"),
+
+    /**
+     * The DataStore is about to start the Sync Queries.
+     */
+    SYNC_QUERIES_STARTED("syncQueriesStarted"),
+
+    /**
+     * All models have been synced.
+     */
+    SYNC_QUERIES_READY("syncQueriesReady"),
```

Comment:
I'm not sure what "sync queries ready" means, just from the name. Is this a good name?

The Javadoc is the same on L63 and L68 so I think at least one needs to be updated.

Reply:
Definitely agree on the naming. That being said, I think we're supposed to align with the names currently used in JS [1]; however, I'll check and see what iOS is currently doing.

Regarding the javadoc, that's my trademarked **Clipboard Inheritance** pattern [2] 🤓 

[1] - https://github.com/aws-amplify/amplify-js/blob/a7feacea4ed506340d250249d0b15286fe3ef5fa/packages/datastore/src/sync/index.ts#L71-L82
[2] - There's a wikipedia page for it, so it's real. https://en.wikipedia.org/wiki/Copy-and-paste_programming

Reply:
This was the one that we talked about in the working group that we could possible rename but if we do it then should be done quickly since the JS versions aren't documented yet. Suggest you sync up with Ivan this week.

Reply:
Fixed the javadoc on this one.